components/tidb_query_vec_expr/src/impl_string.rs

```diff
@@ -3,13 +3,13 @@
 use std::str;
 use tidb_query_codegen::rpn_fn;
 
-use tidb_query_common::Result;
-use tidb_query_datatype::codec::data_type::*;
-use tidb_query_datatype::*;
-use tidb_query_shared_expr::string::{
+use crate::string::{
```

Comment:
How about migrating these things into this file directly?

IMO, no need to keep `conv.rs`, `like.rs`, etc.

Reply:
IMO, conv is big enough to have its own room.